FirstProject/css/feedback.css

```diff
@@ -0,0 +1,162 @@
+/*Feedback HTML*/
+
+.footer_section {
```

Comment:
A best practice would be to make a single class for the footer for example and to apply it in every page to style the footer.(the same for header)

I will try to improve this practice in the future.